be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -815,6 +835,17 @@ bool SegmentIterator::_can_filter_by_preds_except_leafnode_of_andnode() {
     return true;
 }
 
+bool SegmentIterator::_check_apply_by_inverted_index(ColumnId col_id) {
```

Comment:
this function is duplicated with `SegmentIterator::_check_apply_by_inverted_index(ColumnPredicate* pred, bool pred_in_compound)` may be we should put them into one, or rename this function to distinguish them